sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsLiveTests.cs

```diff
@@ -57,10 +57,11 @@ public void ThreadCGUD_ParticipantAUR_MessageGSU_NotificationT()
             var displayNameMessage = "DisplayName sender message 1";
             var participants = new[]
             {
-                new ChatParticipant(user1),
-                new ChatParticipant(user2),
-                new ChatParticipant(user3)
+                new ChatParticipant(user1){ DisplayName = "user1" },
```

Comment:
nit: let's add a space between ) and {
Please apply the same change in the other places too. I see it in other places as well

Reply:
added :)